sources/packages/backend/libs/test-utils/src/factories/notification.ts

```diff
@@ -0,0 +1,58 @@
+import { NotificationEmailMessage } from "@sims/services";
+import {
+  Notification,
+  NotificationMessage,
+  NotificationMessageType,
+  User,
+} from "@sims/sims-db";
+import * as faker from "faker";
+
+/**
+ * Creates a fake message payload.
+ * @returns created message payload.
+ */
+function createDummyMessagePayload(): NotificationEmailMessage {
+  return {
+    email_address: faker.internet.email(),
+    template_id: NotificationMessageType.StudentFileUpload.toString(),
```

Comment:
Template id can be any fake string. Using converting a message type to string seems misleading.